test/configCases/optimization/hashed-module-ids/index.js

```diff
@@ -0,0 +1,10 @@
+var path = require("path");
+
+it("should have named modules ids", function() {
+	for (var i = 1; i <= 5; i++) {
+		var expectedModuleId = "file" + i + ".js";
+		var moduleId = require("./files/file" + i + ".js");
+
+		expect(path.basename(moduleId)).not.toBe(expectedModuleId);
```

Comment:
Too vague. basname doesn't make sense on hash. This would also pass for `hashedModuleIds: false`

Reply:
@sokra how better testing this?

Reply:
/cc @sokra